src/SerialLoops/ViewModels/Dialogs/PreferencesDialogViewModel.cs

```diff
@@ -106,6 +108,19 @@ public void Initialize(PreferencesDialog preferencesDialog, Config config, ILogg
                             RequireRestart = true;
                         },
                     },
+                    new ComboBoxOption([
+                            ("", string.Format(Strings.Default_Font_Display, Strings.Default_Font)),
+                            ..SystemFonts.Collection.Families.Select(_ => (_.Name, _.Name)).ToList(),
```

Comment:
```suggestion
                            ..SystemFonts.Collection.Families.Select(_ => (_.Name, _.Name)),
```
collection expression here means we can use the `IEnumerable` without converting to a `List`, which optimizes slightly